Assignments/SCF-Session-3/string/MyString.java

```diff
@@ -0,0 +1,209 @@
+package string;
+
+import java.util.Scanner;
+
+/**
+ * This class performs operations on strings without using Java APIs.
+ * @author Ishita Mathur
+ *
+ */
+public class MyString 
+{
+	
+	/**
+	 * Calculates length of string
+	 * @param string, string != null
+	 * @return length of string
+	 */
+	public int stringLength(String string)
```

Comment:
`String string` language keywords should not be used for user defined items.

Reply:
done